### PR TITLE
Add support for Sites to SentinelOne threat commands

### PR DESCRIPTION
**Pull request by StephenFerrero**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR aims to add support for filtering by Site ID to the existing get-sentinelone-threats command as well as the fetch command for the SentinelOne integration.

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

**Comment by StephenFerrero** [comment]
I took a look at the CircleCI validations and see the error:

`[IN140] - The integration SentinelOne V2 is currently in skipped. Please add working tests and unskip. Skip comment: License expired issue #24933`

I'm not sure what this means, hopefully someone can point me in the right direction. Thanks

File: Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml [hunk 1]
```diff
@@ -50,6 +50,10 @@ configuration:
   name: fetch_limit
   required: false
   type: 0
+- display: 'Site IDs: comma-separated list of Site IDs to fetch incidents for. (leave blank to fetch all sites)'
```
Comment (ShahafBenYakir) [inline Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.yml:53]:
```suggestion
- display: Site IDs
  additionalinfo: Comma-separated list of Site IDs to fetch incidents for. Leave blank to fetch all sites.
```

**Review by ShahafBenYakir — COMMENTED** [review]
Nice work. 
Don't worry about the failing validation.
Added a small comment on your new parameter - moving the description to the `additionalinfo` key will introduce a nice `?` button the when hovered will show the following description.

**Comment by StephenFerrero** [comment]
Great, I submitted that change, thanks!

**Comment by ShahafBenYakir** [comment]
@StephenFerrero Tried to reach over slack to set up a short demo for this, can you check your DMs?

**Pull request merged**